src/components/search/onSearchBarValueChange.ts

```diff
@@ -0,0 +1,7 @@
+export const onSearchBarValueChange = (state, searchBarValue) => {
+    const value = searchBarValue === undefined || searchBarValue === 0 ? '' : searchBarValue;
```

Comment:
whats the reason for the === check here ?

Reply:
the `||` has higher precedence than `?` so if it's undefined or 0, value should be a blank string

Reply:
oops i misread your question so i deleted my comment haha. we need the strict type checking here because `searchBarValue` is `any` so we don't want things like `0 == '0'` to evaluate to true